### PR TITLE
Landing page

### LINKED ISSUE #533: Create landing page for issues
Create a landing page with a list of open issues with bids for users as a resource to find which issues currently have bids or activity. 

Add Active Issues tab to drop down menu.

### PR DESCRIPTION
**Pull request by jdungan**
Adds a new activity page to the menu so users can see issues with codesy activity.  Also changes the getting started page to show the three most recent issues instead of the github links.

fixes #533.

### CODE REVIEW

**Comment by jdungan** [comment]
New flake8 doesn't like bare excepts.

**Comment by groovecoder** [comment]
I fixed the `flake8` bare except failures in https://github.com/codesy/codesy/pull/511/commits/79f06ee0047e225c5216f60d4a4ffe9243d64945. If we merge that PR first this one should be better too.

File: codesy/templates/includes/get_started.html [hunk 1]
```diff
@@ -39,12 +39,29 @@
...
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Assigned to You" href="https://github.com/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+assignee%3A{{ request.user.username }}"><i class="fi-social fi-social-github medium"></i> Top +1 Issues Assigned to You</a>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Mentioning You" href="https://github.com/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+mentions%3A{{ request.user.username }}"><i class="fi-social fi-social-github medium"></i> Top +1 Issues Mentioning You</a><br/>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Global" href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+"><i class="fi-social fi-social-github medium"></i> Global Top +1 Issues</a><br/>
+            <h4 class="text-left" >You are ready to start bidding.  You can can bid on your own github issues or check out some of the latest activity on codesy:</h4>
```
Comment (groovecoder) [inline codesy/templates/includes/get_started.html]: I think we should keep a "You are ready to bid" header separate from other instructions. Removing the buttons will help draw attention to it, but we want to make it clear to the new user that they are done - they can just use GitHub as normal and codesy can take care of the rest.

An activity table is cool, but let's set it apart from the "You are ready to start bidding. [Go use GitHub like normal.]" header.

**Comment by groovecoder** [comment]
Oops, make that `flake8` fix commit this one: https://github.com/codesy/codesy/pull/511/commits/f8d30f90fab421063c4ccd5c678e1c4a5a5335bf

**Pull request closed without merging**